CODEOWNERS

```
@@ -1 +1 @@
-* @lunarway/squad-nasa
\ No newline at end of file
+* @lunarway/squad-ranger
```

```suggestion
* @bsnlunar 
```

Ranger does not own Open Banking, I would think @bsnlunar is the owner, no?